dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -155,7 +156,12 @@ public static Tool convertEntryToTool(Entry<?, ?> container, DockstoreWebservice
             //TODO: would hook up identified tools that form workflows here
             toolVersion.setIncludedApps(MoreObjects.firstNonNull(toolVersion.getIncludedApps(), Lists.newArrayList()));
             //TODO: hook up snapshot and checksum behaviour here
-            toolVersion.setIsProduction(false);
+            if (version.isFrozen()) {
+                List<ImageData> trsImages = processImageDataForWorkflowVersions(version, toolVersion);
+                toolVersion.setImages(MoreObjects.firstNonNull(trsImages, Lists.newArrayList()));
```

Comment:
It doesn't look like procesImageDataforWorkflowVersions returns a null (nor should it), so this line is unnecessary.